rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -945,7 +1041,7 @@ def __init__(
         self,
         data_signature: Dict[Text, List[FeatureSignature]],
         label_data: RasaModelData,
-        index_tag_id_mapping: Optional[Dict[int, Text]],
+        index_tag_id_mappings: Optional[Dict[Text, Dict[int, Text]]],
```

Comment:
do we need it only to calculate number of tags? Leet's pass `crf_num_tags` directly